README.md

```diff
@@ -1,2 +1,82 @@
-# Documentation-page
-A technical-Documentation-page using html&amp;css
+# Project Title 🚀
+Documentation_page
+## Project Description 📝
+
+> this project is useful for who want to now some basic information about JS.
+Examples:
+
+- When I started learning html and CSS at the first I searched about those.
+- It can give important and basic info about JS.
+- whith this project I learned what is JS and how it is useful.
+
+```html
+<!--  </header>
+    <li class="li">Writing into an alert box, using window.alert().</li>
+    <li class="li">Writing into the browser console, using console.log().</li> -->
+```
+
+```css
+/* 
+.li {
+  font-family: Arial, Helvetica, sans-serif;
+}
+ */
+```
+
+```javascript
+//  class Car {
+        constructor(name, year) {
+          this.name = name;
+          this.year = year;
+        }You can also include JavaScript code snippets.
+```
+
+## Demo 📸
+
+
+![Demo](pic.png)
+
+## Technologies Used 🛠️
+
+- HTML
+- CSS
+
+
+## Installation 💻
+
+Provide step-by-step instructions on how to install and set up your project. Include any dependencies or prerequisites that need to be installed.
+
+```bash
+# git.
```

Comment:
- Add this code here (# git clone git@github.com:latifa-wakili/Documentation-page.git)